aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -1770,22 +1770,22 @@ TEST(NewOperatorRegistrationTest, dispatchAutogradPrecedence) {
 
 TEST(NewOperatorRegistrationTest, throwsWhenRegisterToBackendMapsToAutogradOther) {
   // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-  bool sparsecpu_called, math_called = false;
+  bool fpga_called, math_called = false;
```

Comment:
I had to update these tests to stop using `SparseCPU`. That's because previously, `SparseCPU` was mapped to the `AutogradOther` autograd key. With this change, `SparseCPU`/`QuantizedCPU` are both mapped to `AutogradCPU` (along with the other corresponding backends and their autograd keys). This seems like more rational behavior anyway than what we had before.